Makefile

```diff
@@ -228,3 +228,11 @@ chlog-update: $(CHLOGGEN)
 .PHONY: generate-gh-issue-templates
 generate-gh-issue-templates:
 	$(TOOLS_DIR)/scripts/update-issue-template-areas.sh
+
+.PHONY: check-policies
+check-policies:
+	docker run --rm -v $(PWD)/model:/source -v $(PWD)/docs:/spec -v $(PWD)/policies:/policies \
+		otel/weaver:${WEAVER_VERSION} registry check \
+		--registry=/source \
+		--policy=/policies/registry.rego \
+		--policy=/policies/attribute_name_collisions.rego
```

Comment:
Idea (@lmolkova , @jsuereth): Another option is to move all the Rego files inside the registry itself so they are automatically discovered and applied by `weaver registry check`. Weaver looks at `*.rego` files inside the registry directory by default. To make this more flexible, we could look at `**/*.rego` to retrieve all policy files, even those inside subdirectories of the registry (e.g. `$REGISTRY/policies`). Having the registry YAML and policies under the same folder makes sense to me.

Reply:
I'd prefer to keep the definition and checks separate (I can run these checks on any data whether it's in the semconv repo or not and I can use data without checking it) .

I'd be nice to have any way to pass `*.rego` though - I was going to ask if it's possible.

Reply:
Make sense. I created the following GH issue to keep track of this request. https://github.com/open-telemetry/weaver/issues/259